src/middlewared/middlewared/plugins/vm/utils.py

```diff
@@ -3,6 +3,8 @@
 
 ACTIVE_STATES = ['RUNNING', 'SUSPENDED']
 SYSTEM_NVRAM_FOLDER_PATH = '/data/subsystems/vm/nvram'
+LIBVIRT_QEMU_UID = 64055
+LIBVIRT_QEMU_GID = 64055
```

Comment:
We shouldn't be using IDs this high for service accounts. Please make ticket to shift below ID 1000.

Reply:
Created one https://ixsystems.atlassian.net/browse/NAS-131695 ✅